docs/demo/custom-style.md

```diff
@@ -46,3 +59,25 @@ Using [fenced_divs](https://github.com/jgm/commonmark-hs/blob/master/commonmark-
 
 
 [^mob]: If you are viewing this page on mobile or smaller screens, the embedded notes will be stacked on top of one another because we use Tailwind's [responsive classes](https://tailwindcss.com/docs/responsive-design). Incidentally, we use the `{class=".."}` syntax, rather than the `{.someClass}` syntax, only because the former is [more lenient](https://github.com/jgm/commonmark-hs/issues/76) in accepting non-standard class names, such as the Tailwind responsive classes (eg. `lg:grid-cols-2`).
+
+## External link icons
+
+A link whose address begins with an [URI scheme](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax) component is considered an external link. Emanote displays an icon next to an external hyperlink if the link's description contains some text (the most common non-example are hyperlinks containing only an image). The heuristic is designed to make the site look good in most cases. This behaviour can be overriden by setting the `emanote-external-icon` class to display the icon next to a link containing no text, and by setting the `emanote-external-no-icon` class to prevent displaying an icon next to a link containing text. The classes can also be used if you need to include the icon in the template parts of the site (like the footer or the sidebar) or in raw HTML parts of the document. If this heuristic fails badly in your usecase, please [raise an issue](https://github.com/EmaApps/emanote/issues).
+
+Note that an icon is displayed by default only if, in addition to the scheme component, the address also contains the authority component (i.e. contains `://`) or if the scheme component is `mailto:` or `tel:`. One can easily add custom icons for other URI schemes. Check the CSS in the [`default/templates/base.tpl`](https://github.com/EmaApps/emanote/blob/master/default/templates/base.tpl) template.
+
+* No special styling:
+  * [the emanote repo](https://github.com/EmaApps/emanote)
+  * [why the external link symbol ![[external-link-icon.svg]] is not in Unicode](https://www.unicode.org/alloc/nonapprovals.html)
+  * [$e^{i \pi} + 1 = 0$](https://en.wikipedia.org/wiki/Euler%27s_identity)
+  * [`(>>=) :: forall a b. m a -> (a -> m b) -> m b`](https://hackage.haskell.org/package/base/docs/Prelude.html#v:-62--62--61-)
+  * [![[hello-badge.svg]]](https://emanote.srid.ca)
+  * mother_hetchel@weaversguild.com
+  * [+12 3456789](tel:+123456789)
+* `{.emanote-external-no-icon}` used to suppress the icon:
+  * Water's formula is [H](https://en.wikipedia.org/wiki/Hydrogen){class="emanote-external-no-icon"}₂[O](https://en.wikipedia.org/wiki/Oxygen){class="emanote-external-no-icon"}.
```

Comment:
`{class="emanote-external-no-icon"}` seems too verbose a way to disable the external icon. How about `{data-linktype=none}` or something like that instead? So the CSS query would be something like `a.emanote-external-icon:not([data-linktype/=none])` (assuming a syntax like that is valid).

Reply:
> `{class="emanote-external-no-icon"}` seems too verbose a way to disable the external icon. 

`{.emanote-external-no-icon}` is a bit shorter. The class name could also be shortened, of course. And in principle this is a workaround that shouldn't be used much, as long as the heuristic is good.

> How about `{data-linktype=none}` or something like that instead?

Nice idea. I hadn't thought about using the data-attributes. They are no longer booleans, which has its good and bad sides. I used the data attributes in the new version.